proto/py/compute_image_tools_proto/inspect_pb2.pyi

```diff
@@ -0,0 +1,151 @@
+# @generated by generate_proto_mypy_stubs.py.  Do not edit!
```

Comment:
Why is this file (and all the imports) called inspect_pb2? Why the "2"?

Reply:
The `_pb2` suffix comes from the protoc compiler: https://developers.google.com/protocol-buffers/docs/reference/python-generated